apps/api/src/services/library-traffic.ts

```diff
@@ -0,0 +1,34 @@
+import type { libraryTrafficQuerySchema, libraryTrafficSchema } from "$schema";
+import type { database } from "@packages/db";
+import { eq } from "@packages/db/drizzle";
+import { libraryTraffic } from "@packages/db/schema";
+import type { z } from "zod";
+
+type LibraryTrafficServiceInput = z.infer<typeof libraryTrafficQuerySchema>;
+
+type LibraryTrafficServiceOutput = z.infer<typeof libraryTrafficSchema>;
+
+const transformTrafficRow = (
```

Comment:
why isn't this covered by either zod schema parsing or processing you can do in the SQL query?